powerdnsadmin/routes/index.py

```diff
@@ -652,10 +652,11 @@ def logout():
 
     redirect_uri = url_for('index.login')
     oidc_logout = Setting().get('oidc_oauth_logout_url')
+    id_token_hint = session.get('oidc_token').get('id_token')
```

Comment:
If Local DB access is still enabled and a user goes to logout, they will hit an error as this value is not defined for their session

Reply:
@AgentTNT, can you reproduce the not-defined error? What do you suggest?